.semver

```diff
@@ -0,0 +1,5 @@
+---
+:major: 3
```

Comment:
`master`  versioning now is `4.x`

Reply:
that file should be removed. ignore for now, we can remove after merge as well.